src/transitions/InteractiveArticleLoad.jsx

```diff
@@ -5,8 +5,8 @@ export default class InteractiveArticleLoad extends BaseComponent {
   render() {
     return (
       <div className="interactive-article-load">
-        <h2 className="interactive-article-load__title"></h2>
```

Comment:
The linter was saying this was bad for accessibility due to not having any text in a header, will putting this in a div break it?